src/main/kotlin/com/jetpackduba/gitnuro/ui/dialogs/settings/SettingsDialog.kt

```diff
@@ -383,6 +384,21 @@ fun Logs(settingsViewModel: SettingsViewModel) {
 }
 
 
+@Composable
+private fun Environment(settingsViewModel: SettingsViewModel) {
+    var defaultCloneDir by remember { mutableStateOf(settingsViewModel.defaultCloneDir) }
+
+    SettingTextInput(
+        title = "Default Clone Directory",
```

Comment:
2 details about the text:
- Use string resources for any new strings (The old ones are still being migrated)
- Nitpick: Just "Default" should start with a capital letter